content/docs/getting-started/create-admin-group/index.md

```diff
@@ -33,3 +39,49 @@ Roadie admins have special powers such as the ability to add and remove Backstag
 ![A link that says "Log in to GitHub"](./sidebar-log-into-github.png)
 
 ⚠️  &nbsp;It can take some time for Roadie to refresh the list of teams from GitHub teams. If you do not see admin functions immediately, please wait a few minutes and try again.
+
+
+### Manual admin group configuration with group entities
+
+In case you don't want to install the GitHub app or don't want to manage teams via you can also manually add groups into your Roadie instance.
+
+1. Make a note of the user entity that is assigned to you on the account page of users. You can find this information from `https://<your-tenant>.roadie.so/Account`.
+   * In case you have not logged into GitHub, this is usually the first part of your email address before the `@` symbol.
+
+2. Create a `catalog-info.yaml` entity definition for `roadie-backstage-admin` group
```

Comment:
Where do I create this catalog-info.yaml? 

I wonder if we can make an OS repo for people to fork into their orgs to make this even easier. That can be a separate ticket of course.

Reply:
At this stage I would expect people to be familiar of catalog-info files tbh. An alternative onboarding repo for non-GH is probably is a good idea.